VAMobile/src/store/slices/authSlice.ts

```diff
@@ -79,6 +81,8 @@ export type AuthState = {
   authorizeStateParam?: string
   authParamsLoadingState: AuthParamsLoadingStateTypes
   successfulLogin?: boolean
+  requestNotificationsPreferenceScreen?: boolean
+  requestNotifications?: boolean
```

Comment:
Since this state variable is related to notifications (and also since the authSlice will be going away soon) this should probably be moved to the [notification context](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/de30b5a69ae38c4a3dc3e68f3df98e46448e3e07/VAMobile/src/components/NotificationManager/NotificationManager.tsx#L22-L27)

Reply:
I will do this in the second part of this ticket. Adding an AC.